src/pages/Profile/Profile.jsx

```diff
@@ -0,0 +1,34 @@
+import { useNavigate } from "react-router-dom";
+import { Header } from "../../components/Header/Header";
+import { Tabs } from "../../components/Tabs/Tabs";
+import { useAuth } from "../../context/AuthContext";
+import TabStyles from '../../components/Tabs/Tabs.module.css';
+
+
```

Comment:
you can do one thing, as you are importing from the components why not to make a single file and import from there.
[read here](https://dev.to/justmyrealname/organize-react-components-better-with-barrel-exports-4fno)

Reply:
okeiy will do during refactor. thanks for suggestion.